drakrun/drakrun/main.py

```diff
@@ -372,6 +353,38 @@ def process(self):
                 dump_dir = os.path.join(outdir, "dumps")
                 drakmon_log_fp = os.path.join(outdir, "drakmon.log")
 
+                injector_cmd = ["injector",
+                                "-o", "json",
+                                "-d", "vm-{vm_id}".format(vm_id=INSTANCE_ID),
+                                "-r", kernel_profile,
+                                "-i", inject_pid,
+                                "-k", kpgd,
+                                "-m", "writefile",
+                                "-e", f"%USERPROFILE%\\Desktop\\{file_name}",
+                                "-B", os.path.join(workdir, file_name)]
+
+                self.log.info("Running injector...")
+                injector = subprocess.Popen(injector_cmd, stdout=subprocess.PIPE)
+                outs, errs = injector.communicate(b"", 20)
+
+                if injector.returncode != 0:
+                    raise subprocess.CalledProcessError(injector.returncode, injector_cmd)
+
+                injected_fn = json.loads(outs)['ProcessName']
+                net_enable = int(self.config.config['drakrun'].get('net_enable', '0'))
```

Comment:
`self.config.config['drakrun']` ugly :donotwant:

Reply:
This is already used this way in multiple places within this file. I agree this needs refactor but this is out of scope of this PR